### DISCUSSION TITLE
[Idea] Partial inventory of changes controlled by GLPI-Server / Proxy Server

### DISCUSSION THREAD

**Discussion by erique-souza** [glpi-project/glpi-agent#793] [Ideas]
### Your idea

The full-inventory-postpone system is fantastic, congratulations to the developers and whoever came up with the idea, it really helps an extensive IT park that needs frequent updating, I recently opened an issue reporting what I believe to be a bug, which I haven't seen the function itself being functional when trying to edit its default number https://github.com/glpi-project/glpi-agent/issues/792

It's not something serious, but it's a point of observation, in any case the suggestion in this topic would be that in addition to being allowed to change this "full-inventory-postpone" parameter directly in each agent, it is also possible to configure it in the glpi itself. server and also in the agent's own proxy service, having a priority level, where if configured on the server/proxy it will overwrite the information that could be on the agent individually

The idea is to have greater control manageable by the glpi server itself and of course when using the proxy as well

**Comment by g-bougard** [comment]
This is a good idea and to be honest this is still something I want to see myself... but indeed for the all glpi-agent configuration: my dream is to have a glpi-agent registration process and have the server to push the final configuration with the availability to update it at any time. Actually this is not a plan as it required some devs server-side and the team is focused on GLPI 11. I hope we will have a project a such a subject next year.
